include/meson.build

```diff
@@ -0,0 +1 @@
+incdir = include_directories('.', './msg')
```

Comment:
This is incorrect. It would allow includes in the library such as `#include <msg.h>` instead of the intended `#include <msg/msg.h>`.